### PR TITLE
don't merge: code-review

### PR DESCRIPTION
**Pull request by Sumeetrana**
**PLEASE DONT MERGE. Its only  to review comments**

### CODE REVIEW

File: cw-rct201/src/App.tsx [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-      <AllRoutes />
-      <ToastContainer/>
-       <Footer />
-      
-
-      
-    
-    
-
-
```
Comment (Sumeetrana) [inline cw-rct201/src/App.tsx:29]: remove extra spaces

File: cw-rct201/src/App.tsx [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-function App() {
-  return (
-    <div className="App">
-
-   
-      <Navbar />
-      <AllRoutes />
-      <ToastContainer/>
-       <Footer />
```
Comment (Sumeetrana) [inline cw-rct201/src/App.tsx:22]: Fix indentation for this file

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx [hunk 3]
```diff
@@ -1,518 +0,0 @@
...
-  const [alertDeleteOpen, setDeleteOpen] = React.useState<boolean>(false);
-  const [newName, setnewName] = React.useState<string>("");
-  const [newCat, setNewCat] = React.useState<string>("");
-  const [newPrice, setNewPrice] = React.useState<any>();
-  const [categoryDrawerOpen, setCategoryDrawerOpen] = React.useState<boolean>(false);
-
-  const [newImage, setNewImage] = React.useState<string>("");
-  const [image1DrawerOpen, setImage1DrawerOpen] = React.useState<boolean>(false);
-
-  const cancelRef = React.useRef<any>();
```
Comment (Sumeetrana) [inline cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:64]: add comment above this to tell what this 'cancelRef' is doing and why is it used

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx [hunk 4]
```diff
@@ -1,518 +0,0 @@
...
-}
-const imageUpdate=()=>{
-  toast.success("Image Updated Successfully",{theme:"colored"})
-}
-  const updateName = (id: number) => {
-    let obj = {
-      name: newName,
-    };
-
-    axios.patch(`https://backendsirver-for-daily-needs.vercel.app/products/${id}`, obj);
```
Comment (Sumeetrana) [inline cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:85]: Fix indentation for this file as well as for the whole project

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx [hunk 5]
```diff
@@ -1,518 +0,0 @@
...
-    setTimeout(()=>{
-      // dispatch(sendDataToRedux());
-    },1500)
-    
-   
-    nameSuccess()
-  };
-  const updatePrice = (id: number) => {
-    let obj = {
-      cost: +newPrice,
```
Comment (Sumeetrana) [inline cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:98]: why are you sending +newPrice in response? what is it determining? please add comments above this line

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx [hunk 6]
```diff
@@ -1,518 +0,0 @@
...
-      {" "}
-      Category : {category}
-    </Text>
-   
-    <Text mt={1} fontWeight={"bold"} fontSize={"16px"}>
-      Price : ₹ {cost}
-    </Text>
-    </Box>
-              </Flex>
-              {/* <SimpleGrid  alignItems={"center"} border="1px solid red" margin="auto" textAlign="center" columns={2}>
```
Comment (Sumeetrana) [inline cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:357]: Remove unnecessary comments

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx [hunk 7]
```diff
@@ -1,518 +0,0 @@
...
-
-      {/* This is wehre we give users an option to do update name,price,category */}
-       
-      </>
-    </Box>
-    </>
-  );
-};
-
-export default AdminSingleProduct;
```
Comment (Sumeetrana) [inline cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:483]: This file is too big. Try to break down this file into a few smaller components. The ideal size of a component is 100 lines. If component is becoming lengthier than that, then break it down into smaller components.

File: cw-rct201/src/Components/Admin/Users.tsx [hunk 8]
```diff
@@ -1,124 +0,0 @@
...
-      setTimeout(() => {
-        dispatch(getUsersData());
-     
-      }, 1500);
-      deleteSuccess();
-    } catch (error) {
-      console.log("error ", error);
-    }
-  };
-  console.log(users, "users arr");
```
Comment (Sumeetrana) [inline cw-rct201/src/Components/Admin/Users.tsx:56]: Remove console logs

**Review by Sumeetrana — COMMENTED** [review]
**PLEASE DON"T MERGE THIS**

File: cw-rct201/src/App.tsx (hunk 1 above)
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/App.tsx:29]: Fixed Sir

File: cw-rct201/src/App.tsx (hunk 2 above)
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/App.tsx:22]: Fixed Indentation sir and now its  formatted by prettier also

File: cw-rct201/src/Components/Admin/AddProduct.tsx [hunk 9]
```diff
@@ -1,127 +0,0 @@
...
-  Text,
-  useColorModeValue,
-  Select,
-} from "@chakra-ui/react";
-import axios from "axios";
-import React from 'react'
-import { toast } from "react-toastify";
-
-type product={
-name:string;
```
[the remark this answers is not in the document]
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Admin/AddProduct.tsx:18]: Fixed Sir

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx (hunk 3 above)
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:64]: added comment sir why and what about cancelRef

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx (hunk 4 above)
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:85]: Fixed indentation sir

Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:85]: Added Comments sir why is the +new price

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx (hunk 6 above)
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:357]: Removed Unnecessary comments sir

File: cw-rct201/src/Components/Admin/Users.tsx (hunk 8 above)
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Admin/Users.tsx:56]: Fixed Sir

File: cw-rct201/src/Components/Checkout/Checkout.tsx [hunk 10]
```diff
@@ -1,324 +0,0 @@
...
-                      fill="#d50000"
-                      d="M16 10A14 14 0 1 0 16 38A14 14 0 1 0 16 10Z"
-                    />
-                    <path
-                      fill="#ff3d00"
-                      d="M18,24c0,4.755,2.376,8.95,6,11.48c3.624-2.53,6-6.725,6-11.48s-2.376-8.95-6-11.48 C20.376,15.05,18,19.245,18,24z"
-                    />
-                  </svg>
-
-                  <svg
```
[the remark this answers is not in the document]
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Checkout/Checkout.tsx:67]: Done Sir

File: cw-rct201/src/Components/Homepage/Homepage.tsx [hunk 11]
```diff
@@ -1,67 +0,0 @@
...
-  React.useEffect(() => {
-    dispatch(getUsersData())
-    
-  }, []);
-  return (
-    <div >
-        <div>
-          
-          <img  className='front-flex-img' src="https://images.dailyobjects.com/marche/assets/images/other/republicsale-home-page-desktop.png?tr=cm-pad_resize,v-2,w-1351,dpr-1" alt="" />
-
```
[the remark this answers is not in the document]
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Homepage/Homepage.tsx:38]: Fixed Sir

File: cw-rct201/src/Pages/Admin.tsx [hunk 12]
```diff
@@ -1,270 +0,0 @@
...
-import { Data } from "./MobileProducts";
-import { userobj } from "./Signup";
-
-const Admin = () => {
-
-
-  const [totalAdminArr,setTotalAdminarr]=React.useState<userobj[]>([])
-
-
-
```
[the remark this answers is not in the document]
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Pages/Admin.tsx:40]: Removed From All Project sir

File: cw-rct201/src/Pages/Signup.tsx [hunk 13]
```diff
@@ -1,232 +0,0 @@
...
-  };
-
-  const [checkUser, setCheckUsers] = useState<userobj[]>([]);
-  const getData = async () => {
-    let allLoginUsers = await loginApi();
-
-    setCheckUsers(allLoginUsers);
-  };
-
-  const makeDisable = () => {
```
[the remark this answers is not in the document]
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Pages/Signup.tsx:66]: Renamed sir

File: cw-rct201/src/Pages/SingleProduct.tsx [hunk 14]
```diff
@@ -1,483 +0,0 @@
...
-      setdata(d);
-    } catch (error) {
-      console.log(error);
-    }
-  };
-  // console.log(id)
-  useEffect(() => {
-    getdata();
-
-  getUserData()
```
[the remark this answers is not in the document]
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Pages/SingleProduct.tsx:105]: Fixed sir

File: cw-rct201/src/Pages/SingleProduct.tsx [hunk 15]
```diff
@@ -1,483 +0,0 @@
...
-  }, []);
-
-  const getUserData = async () => {
-    try {
-      let r = await fetch(`https://backendsirver-for-daily-needs.vercel.app/users/${currentUser.id}`);
```
[the remark this answers is not in the document]
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Pages/SingleProduct.tsx:110]: Given a Meaningful name sir

File: cw-rct201/src/Components/Admin/AdminSingleProduct.tsx (hunk 7 above)
Comment (HANUMAT-SHARAN) [reply cw-rct201/src/Components/Admin/AdminSingleProduct.tsx:483]: Yes Sir I have Break the Fille into Smaller Components Now

**Comment by HANUMAT-SHARAN** [comment]
Thank you So Much Sir For Your Valuable Feedback I have learned Allot Of new things from the feedback that you have provided

**Pull request closed without merging**